Project.toml

```diff
@@ -1,13 +1,13 @@
 name = "FFMPEG"
 uuid = "c87230d0-a227-11e9-1b43-d7ebe4e7570a"
 authors = ["SimonDanisch <sdanisch@gmail.com>"]
-version = "0.4.1"
+version = "0.5.0"
 
 [deps]
 FFMPEG_jll = "b22a6f82-2f65-5046-a5b2-351ab43fb4e5"
 
 [compat]
-FFMPEG_jll = "4.3.1"
+FFMPEG_jll = "6.1.1"
```

Comment:
Can we keep both, not to break too dramatically packages which still need v4?

Reply:
They would still need to update their FFMPEG.jl to 0.5.0 right?
Should I then change the bump of FFMPEG.jl to 0.4.2?

Reply:
Another issue is that 6.1.1 needs julia 1.6, I guess your suggestion is then necessary.